.devcontainer/devcontainer.json

```diff
@@ -26,5 +26,8 @@
 	// "postCreateCommand": "gcc -v",
 
 	// Comment out to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
-	"remoteUser": "vscode"
+	"remoteUser": "vscode",
+	"mounts": [
+		"source=C:\\ppm_data,target=/ppm_data,type=bind,consistency=cached"
```

Comment:
Is this assuming a C:\ level directory of "ppm_data"?

Reply:
Yup! Should we add a note to the README?

Reply:
Can we configure this to a relative path for the ppm module? Not everyone will have a C drive, let alone a ppm_data directory on it

Reply:
We need to mount a directory that contains fieldsToRedact.txt to the dev container when we spin it up or else the tests will fail when running them in the container, but it doesn't have to be from the C drive. I'll look into pulling the location of the directory from a specified location.

Reply:
There will be some users that don't require anything to be redacted. Can we make it optional?

Reply:
Yup! The fieldsToRedact.txt file has been made optional.

Reply:
It turns out that relative paths cannot be used for directory mounts in devcontainer.json.

Reply:
This mount has been removed.